english/you-are-in-minecraft/dig_down/dig_down.md

```diff
@@ -24,4 +24,6 @@ Do you go [left?](goleft/goleft.md)
 
 Or maybe go [right?](goright/goright.md) 
 
+Do you continue to [dig down] (dig_down_more)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.